sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -309,6 +309,41 @@ You should see output similar to:
     node.js version     :  v6.10.3
     npm version         :  3.10.10
 
+The next check that can be made is to generate one of the example figures.
+There are a examples for the different ways in which bokeh can be used.
+
+To create an html file,
+
+.. code-block:: sh
+
+    env BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
```

Comment:
This is `csh`/`tcsh` specific , need to demonstrate also with Bash syntax (which I also think should be first) 
```
BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
```
as well as for Windows (I don'r recall what that looks like offhand)